### PR TITLE
Update zkSync Layer 2 purpose to "universal", description, and Logo

### PR DESCRIPTION
**Pull request by zk-Lumi**
## Description

Updated zkSync purpose to "universal" + update description to meta description of website

### CODE REVIEW

**Comment by zk-Lumi** [comment]
> Hey @zk-Lumi, appreciate this! To my understanding there is now zkSync Lite, and zkSync Era... the first one was the legacy one which was where this entry came from, and still exists (though seems to be transitioning out)
> 
> image image
> If we're going to update this to use zkSync Era information, I think we should also update the name and logo to match. Let me know if I'm missing something or if you disagree.

Hi @wackerow,

For logo: Good heads up, I updated the logo to the newest branding. 
For name: Lite is indeed being transitioned out and "Era" branding may or may not be phased out, so be good to keep name as "zkSync" for now.

**Comment by zk-Lumi** [comment]
> Hey @zk-Lumi, appreciate this! To my understanding there is now zkSync Lite, and zkSync Era... the first one was the legacy one which was where this entry came from, and still exists (though seems to be transitioning out)
> 
> image image
> If we're going to update this to use zkSync Era information, I think we should also update the name and logo to match. Let me know if I'm missing something or if you disagree.

Hi @wackerow,

For logo: Good point, I updated the logo to the newest branding.
For name: Lite is indeed being transitioned out and "Era" branding may or may not be phased out, so be good to keep name as "zkSync" for now.

**Comment by zk-Lumi** [comment]
Ping on this @wackerow 🙏

File: src/intl/en/page-layer-2.json [hunk 1]
```diff
@@ -124,7 +124,7 @@
   "boba-description": "Boba is an Optimistic Rollup originally forked from Optimism which is a scaling solution that aims to reduce gas fees, improve transaction throughput, and extend the capabilities of smart contracts.",
   "base-description": "Base is a secure, low-cost, developer-friendly Ethereum L2 built to bring the next billion users to web3. It is an Ethereum L2, incubated by Coinbase and built on the open-source OP Stack.",
   "loopring-description": "Loopring's zkRollup L2 solution aims to offer the same security guarantees as Ethereum mainnet, with a big scalability boost: throughput increased by 1000x, and cost reduced to just 0.1% of L1.",
-  "zksync-description": "zkSync is a user-centric zk rollup platform from Matter Labs. It is a scaling solution for Ethereum, already live on Ethereum mainnet. It supports payments, token swaps and NFT minting.",
+  "zksync-description": "zkSync is a ZK Rollup that aims to scale Ethereum and its values to mainstream adption, without degrading security or decentralization.",
```
Comment (wackerow) [inline src/intl/en/page-layer-2.json:127]:
```suggestion
  "zksync-description": "zkSync is a ZK Rollup that aims to scale Ethereum and its values to mainstream adoption, without degrading security or decentralization.",
```

**Review by wackerow — APPROVED** [review]
Looks good! Thanks @zk-Lumi!

**Comment by wackerow** [comment]
@all-contributors please add @zk-Lumi for maintenance

**Pull request merged**